test/testvectorsgen/go.mod

```diff
@@ -5,7 +5,10 @@ go 1.13
 replace github.com/iden3/go-iden3-core => ../../../go-iden3-core
```

Comment:
Now that the PR https://github.com/iden3/go-iden3-core/pull/374 is merged, this can be removed, right?

Reply:
yes!